core/src/com/mygdx/game/Components/Pirate.java

```diff
@@ -80,10 +80,10 @@ public void shoot(Vector2 dir) {
     /**
      * Adds ammo
      *
-     * @param ammo amount to add
+     * @param newAmmo amount to add
```

Comment:
Why is the variable Ammo changed?

Reply:
There was a conflict with using the same variable name 'ammo' for the new ammo to be added to the existing ammo, as changing the function to static meant not being able to use the "this" reference.

Issue simply resolved by not naming them the same thing.